docs/Platform-Architecture.md

```diff
@@ -1,4 +1,4 @@
-# TaskManager
+# Platform Architecture
```

Comment:
Another thing to consider would be moving the `Issuing Commands` section to a different document, as it is of interest to the users and `Platform-Architecture.md` is placed under Developer documentation (this can also be done in a subsequent PR).

Reply:
Yes, I'll file an issue for that.